src/scenes/Wallet/components/Balances/styles.js

```diff
@@ -0,0 +1,62 @@
+const styles = (theme) => ({
+  rootPaper: {
+    boxShadow: 'none',
+    borderRadius: theme.borderRadius,
+    background: theme.palette.background.paper,
+  },
+  containerGrid: {
+    padding: theme.padding.lg.px,
+    overflowX: 'hidden',
+  },
+  myBalanceTitle: {
+    marginBottom: theme.padding.xs.px,
+  },
+  totalsContainerGrid: {
+    marginBottom: theme.padding.sm.px,
+  },
+  totalsItemGrid: {
+    marginRight: theme.padding.md.px,
+  },
+  totalsItemAmount: {
+    fontSize: theme.sizes.font.titleMd,
+    marginBottom: 2,
+  },
+  table: {
+    border: theme.border,
+  },
+  tableHeader: {
+    height: 40,
+    background: theme.palette.background.grey,
+  },
+  tableHeaderItemText: {
+    fontWeight: theme.typography.fontWeightBold,
+  },
+  tableRow: {
+    height: 42,
```

Comment:
for commonly used numbers like this, we want to add a var in theme object like theme.sizes.tableRowHeight: 40

Reply:
according to the sketches, I've seen three diff tableRowHeights so not exactly sure if there is a standard height yet. in this case, I feel like we should use padding instead and let the contents dictate the height. what do you think?